pan3d/__main__.py

```diff
@@ -1,17 +1,18 @@
-from argparse import ArgumentParser
+import argparse
 from dataset_builder import DatasetBuilder
 
-parser = ArgumentParser(
+parser = argparse.ArgumentParser(
     prog="Pan3D",
     description="Launch the Pan3D Dataset Builder",
 )
 
 parser.add_argument("-b", "--config_path")
 parser.add_argument("-d", "--dataset_path")
+parser.add_argument("-s", "--server", action=argparse.BooleanOptionalAction)
```

Comment:
Does trame already take care of that?

Reply:
That is because you don't use the CLI from trame... But your approach is probably better if we really make trame optional for the viewer section down the road...